spring-petclinic/pom.xml

```diff
@@ -748,21 +443,21 @@
 			<plugin>
```

Comment:
I am not sure what to do about this. The Pivitol version has this plugin. I only set it to how we do it. I can leave the whole thing running on Java 7 and then it would really match upstream. What do you suggest?